hybridse/include/sdk/result_set.h

```diff
@@ -79,7 +79,7 @@ class ResultSet {
                 return std::to_string(GetDoubleUnsafe(idx));
             }
             case kTypeBool: {
-                return std::to_string(GetBoolUnsafe(idx));
+                return GetBoolUnsafe(idx) ? "true" : "false";
```

Comment:
@jingchen2222 there are any effects on hybridse if modify the bool value in `GetAsString`?

Reply:
`GetAsString` is used by the client. It's used only for viewing.
I think it's ok to modify the bool value's string format.